src/functions/schema.ts

```diff
@@ -1,21 +1,34 @@
 import * as AJV from 'ajv';
-import * as jsonSpecv4 from 'ajv/lib/refs/json-schema-draft-04.json';
-const oasFormatValidator = require('ajv-oai/lib/format-validator');
 import { ValidateFunction } from 'ajv';
+import * as jsonSpecv4 from 'ajv/lib/refs/json-schema-draft-04.json';
 import { IOutputError } from 'better-ajv-errors';
 import { JSONSchema4, JSONSchema6 } from 'json-schema';
 import { IFunction, IFunctionResult, ISchemaOptions } from '../types';
+const oasFormatValidator = require('ajv-oai/lib/format-validator');
 const betterAjvErrors = require('better-ajv-errors/lib/modern');
 
 interface IAJVOutputError extends IOutputError {
   path?: string;
 }
 
+const logger = {
+  warn(...args: any[]) {
+    const firstArg = args[0];
+    if (typeof firstArg === 'string') {
```

Comment:
I believe we want to hide those messages only for oas2/oas3 documents, don't we?

Reply:
@P0lip I don't think so. I think we want to hide for everyone trying to validate JSON Schema. According to the spec unknown values are OK.

Reply:
https://github.com/epoberezkin/ajv/issues/466#issuecomment-296290767 this was posted by Phil in the issue.
I agree with that. In general, we do want to log that. If we don't, then the fix is just a one-liner -> `unknownFormats: 'ignore' :P

Reply:
@P0lip `unknownFormats: ignore` does not suppress logging it replaces errors with `warn` logs.

Reply:
See that we already had this option turned on

```
const ajv = new AJV({
  meta: false,
  schemaId: 'auto',
  jsonPointers: true,
  unknownFormats: 'ignore',
  logger,
});
```
https://github.com/stoplightio/spectral/blob/94b91c32a5b854d6a896d9aee396347447ad450b/src/functions/schema.ts#L30

Reply:
I know we had it set up, I just thought there is a `false` option lol, but there isn't any :D